Dockerfile

```diff
@@ -13,10 +13,8 @@ LABEL \
     version="$commitid"
 
 
-# An internal yum repo is needed for rhmsg
-ADD http://download.devel.redhat.com/rel-eng/RCMTOOLS/rcm-tools-fedora.repo /etc/yum.repos.d/
-# ...but the image doesn't have the required root CA installed
-RUN sed -i 's_https://_http://_' /etc/yum.repos.d/rcm-tools-fedora.repo
+# Use Copr repo for python3-rhmsg package
+ADD https://copr.fedorainfracloud.org/coprs/qwan/python-rhmsg/repo/fedora-32/qwan-python-rhmsg-fedora-32.repo /etc/yum.repos.d/
```

Comment:
Instead of this, you can use "RUN dnf install -y 'dnf-command(copr)' && dnf copr enable -y qwan/python-rhmsg", then when we update our image to latest fedora, we don't need to make any change to this.

Reply:
> In addition to this change, you need to update the related part of installing ca cert, because we now build the image in quay.io and it can not access internal network. You can install ca cert in container's entrypoint script.

And what script is entrypoint? In Dockerfile it says **fedms-hub-3**, and as I understand it will start consumer, but where this configuration located?

Reply:
@apaplaus I think @qixiang is suggesting that you create a wrapper script  that does this and then runs `exec fedmsg-hub-3` at the end. Greenwave does this:
https://pagure.io/greenwave/blob/master/f/docker/docker-entrypoint.sh

My suggestion is to remove this functionality entirely and just mount the certificate bundle with the internal CAs in the container image at runtime. We do this for many deployments already.

I'll leave it up to you to chose which route you prefer.